### PR TITLE
[Keyboard] Add support S6xty5

### PR DESCRIPTION
**Pull request by trnthsn**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/trnthsn/s6xty5/chconf.h
Comment (zvecr) [inline keyboards/trnthsn/s6xty5/chconf.h:1]: This file should be removed.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/trnthsn/s6xty5/chconf.h
Comment (trnthsn) [reply keyboards/trnthsn/s6xty5/chconf.h:1]: i deleted this file

**Review by drashna — APPROVED** [review]

File: keyboards/trnthsn/tyson60s/keyboard.json
Comment (fauxpark) [inline keyboards/trnthsn/tyson60s/keyboard.json:1]: Please split changes to this board into a separate PR. It will also need to go through `develop`.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/trnthsn/tyson60s/keyboard.json
Comment (trnthsn) [reply keyboards/trnthsn/tyson60s/keyboard.json:1]: Thank you, i pushed new commit.

File: keyboards/trnthsn/s6xty5/chconf.h
Comment (trnthsn) [reply keyboards/trnthsn/s6xty5/chconf.h:1]: @zvecr could you review my PR again

File: keyboards/trnthsn/s6xty5/keyboard.json [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+    },
+    "layout_aliases": {
+        "LAYOUT": "LAYOUT_all"
+    },
+    "community_layouts": [
+        "65_ansi_blocker",
+        "65_ansi_blocker_split_bs",
+        "65_ansi_blocker_tsangan",
+        "65_ansi_blocker_tsangan_split_bs"
+    ],
```
Comment (dunk2k) [inline keyboards/trnthsn/s6xty5/keyboard.json:59]: 1. layout aliases not required
2. community layout one lined
```suggestion
    "community_layouts": ["65_ansi_blocker", "65_ansi_blocker_split_bs", "65_ansi_blocker_tsangan", "65_ansi_blocker_tsangan_split_bs"],
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/trnthsn/s6xty5/keyboard.json (hunk 1 above)
Comment (trnthsn) [reply keyboards/trnthsn/s6xty5/keyboard.json]: I updated

File: keyboards/trnthsn/s6xty5/keyboard.json [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+    },
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
```
Comment (fauxpark) [inline keyboards/trnthsn/s6xty5/keyboard.json:27]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/trnthsn/s6xty5/keyboard.json (hunk 2 above)
Comment (trnthsn) [reply keyboards/trnthsn/s6xty5/keyboard.json]: @fauxpark thank you. I updated my keyboard.json

**Comment by trnthsn** [comment]
Could you please review my PR again, @zvecr and @fauxpark?

**Review by fauxpark — APPROVED** [review]

File: keyboards/trnthsn/s6xty5/keyboard.json [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
+                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"matrix": [4, 6], "x": 6, "y": 4, "w": 1.25},
+                {"matrix": [4, 10], "x": 10, "y": 4, "w": 1.25},
+                {"matrix": [4, 11], "x": 11.25, "y": 4, "w": 1.25},
+                {"matrix": [4, 12], "x": 13, "y": 4},
+                {"matrix": [4, 13], "x": 14, "y": 4},
+                {"matrix": [4, 14], "x": 15, "y": 4}
+            ]
+        },
```
Comment (dunk2k) [inline keyboards/trnthsn/s6xty5/keyboard.json:126]:
```suggestion
    "layout_aliases": {
        "LAYOUT_all": "LAYOUT_65_ansi_blocker_split_bs"
    },
    "community_layouts": ["65_ansi_blocker", "65_ansi_blocker_split_bs", "65_ansi_blocker_tsangan", "65_ansi_blocker_tsangan_split_bs"],
    "layouts": {
```

File: keyboards/trnthsn/s6xty5/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    // ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┼───┤
+    // │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │  Enter │PgD│
+    // ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┼───┤
+    // │ Shift  │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │ Shift│ ↑ │End│
+    // ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬─┬───┼───┼───┤
+    // │Ctrl│LGUI│Alt │                        │RGUI│ Fn │ │ ← │ ↓ │ → │
+    // └────┴────┴────┴────────────────────────┴────┴────┘ └───┴───┴───┘
+
+
+    [0] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/trnthsn/s6xty5/keymaps/default/keymap.c:20]:
```suggestion
    [0] = LAYOUT_all(
```

File: keyboards/trnthsn/s6xty5/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSPC, KC_DEL,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,  KC_PGDN,
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RGUI, MO(1),   KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/trnthsn/s6xty5/keymaps/default/keymap.c:27]:
```suggestion
    [1] = LAYOUT_all(
```

File: keyboards/trnthsn/s6xty5/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_TRNS,  KC_DEL, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        MO(2)  , KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_RGHT
+    ),
+    [2] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/trnthsn/s6xty5/keymaps/default/keymap.c:34]:
```suggestion
    [2] = LAYOUT_all(
```

File: keyboards/trnthsn/s6xty5/config.h [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define WS2812_SPI_DRIVER SPID2
+#define WS2812_SPI_MOSI_PAL_MODE 0
+#define WS2812_SPI_SCK_PAL_MODE 0
+#define WS2812_SPI_SCK_PIN B13
```
Comment (dunk2k) [inline keyboards/trnthsn/s6xty5/config.h:23]:
```suggestion
#define WS2812_SPI_SCK_PIN B13

```

File: keyboards/trnthsn/s6xty5/halconf.h [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define HAL_USE_SPI TRUE
+
+#include_next <halconf.h>
+
```
Comment (dunk2k) [inline keyboards/trnthsn/s6xty5/halconf.h:22]:
```suggestion
```

File: keyboards/trnthsn/s6xty5/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+        QK_BOOT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_RGHT
+    ),
+};
```
Comment (dunk2k) [inline keyboards/trnthsn/s6xty5/keymaps/default/keymap.c:41]:
```suggestion
};

```

**Review by dunk2k — CHANGES_REQUESTED** [review]
1. Replace LAYOUT in `layouts` with `layout_aliases`; to conform to guidance regarding multi layout board/PCB
    1.  change layout name in `default/keymap.c` to match
2. Tidy End of File lines

**Comment by trnthsn** [comment]
> 1. Replace LAYOUT in `layouts` with `layout_aliases`; to conform to guidance regarding multi layout board/PCB
>    
>    1. change layout name in `default/keymap.c` to match
> 2. Tidy End of File lines

Thank you, i applied your suggestion
